src/armonik_cli/commands/sessions.py

```diff
@@ -0,0 +1,206 @@
+import grpc
+import rich_click as click
+
+from datetime import timedelta
+from typing import List
+
+from armonik.client.sessions import ArmoniKSessions
+from armonik.common import SessionStatus, Session, TaskOptions
+
+from armonik_cli import console, errors
+from armonik_cli.commands.common import endpoint_option, output_option, debug_option
+
+
+SESSION_TABLE_COLS = [("ID", "SessionId"), ("Status", "Status"), ("CreatedAt", "CreatedAt")]
+
+
+@click.group(name="sessions")
+def sessions():
+    """Manage cluster sessions."""
+    pass
+
+
+@sessions.command()
+@endpoint_option
+@output_option
+@debug_option
+@errors.error_handler
+def list(endpoint: str, output: str, debug: bool):
+    """List the sessions of an ArmoniK cluster."""
```

Comment:
docstring

Reply:
All the documentation is already provided in the argument and option definitions. Writing the docstrings lead to repeat a lof of things!